boxsdk/object/trash.py

```diff
@@ -101,14 +113,43 @@ def get_items(self, limit: Optional[int] = None, offset: Optional[int] = None, f
             The offset of the item at which to begin the response.
         :param fields:
             List of fields to request.
+        :param sort:
+            The field to sort by.
+        :param direction:
+            The direction to sort.
+        :param marker:
+            The marker at which to begin the response.
+        :param use_marker:
+            Whether or not to use marker-based paging.
         :returns:
             An iterator of the entries in the trash
         """
+        additional_params = {}
+        if limit is not None:
+            additional_params['limit'] = limit
+        if direction is not None:
```

Comment:
maybe just `if not direction:`. Empty string is also not valid in this case. Same for limit if statement